include/boost/geometry/algorithms/detail/disjoint/multipoint_geometry.hpp

```diff
@@ -310,23 +317,27 @@ class multi_point_multi_geometry
         }
     };
 
+    template <typename DisjointPointBoxStrategy>
     struct overlaps_box_point
     {
         template <typename Box, typename Point>
         static inline bool apply(Box const& box, Point const& point)
         {
             // The default strategy is enough for Point/Box
```

Comment:
Maybe this comment can be removed now (is it still the default strategy? It can now be specified)

Reply:
Yes, it should have been removed.